src/commands/app/create.ts

```diff
@@ -236,8 +259,8 @@ export default class Create extends BaseCommand<typeof Create> {
   manageManifestToggeling() {
     // NOTE Use boilerplate manifest if exist
     const manifestPath = resolve(
-      this.sharedConfig.folderPath || "",
-      "manifest.json"
+      this.sharedConfig.folderPath,
+      `${this.sharedConfig.folderPath}/manifest.json`
     );
```

Comment:
@harshithad0703 , add check if manifest file not present in repo then use static one.

Reply:
the existSync will check and add the the static manifest.json